app/index.js

```diff
@@ -2,21 +2,21 @@ import React, { View, Text } from 'react-native';
 
 import styles from './styles/default';
 
-class TwoByTwoCards extends React.Component {
+const TwoByTwoCards = React.createClass({
```

Comment:
**<a href='http://eslint.org/docs/rules/object-curly-spacing'>object-curly-spacing</a>**: A space is required after '{'